src/aliases/api/handlers/utils.go

```diff
@@ -0,0 +1,68 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+)
+
+func jsonWrite(w http.ResponseWriter, data interface{}) error {
```

Comment:
To return a json object in the http response we are using the following

```
import "encoding/json"
...
_ = json.NewEncoder(rw).Encode(formatters.FormatSecretResponse(secret))
```

Reply:
It's mostly to have a common way to add the HTTP headers homogenically.
In the end, it just does what you say as well.

Reply:
As we agree, we would need to move this into `/src/infra/http`

Reply:
done